tsconfig.json

```diff
@@ -2,7 +2,7 @@
   "compilerOptions": {
     "target": "ES2018",
     "module": "commonjs",
-    "lib": ["es2018"],
+    "lib": ["es2018", "DOM"],
```

Comment:
What does this do?

Reply:
This is to mitigate https://github.com/microsoft/TypeScript-DOM-lib-generator/issues/826

Esbuild requires the WASM module